zammad/templates/statefulset.yaml

```diff
@@ -99,6 +101,8 @@ spec:
           volumeMounts:
             - name: {{ template "zammad.fullname" . }}-var
               mountPath: /opt/zammad/var
+            - name: {{ template "zammad.fullname" . }}-var
```

Comment:
Is it intended that you use the same name `...-var` twice?

Reply:
I just followed your code as you already have 
https://github.com/zammad/zammad-helm/blob/c9c243e36150743db002ed51a98bed9d10919c8e/zammad/templates/statefulset.yaml#L408